docs/Developers/Linter_Bears_Advanced.rst

```diff
@@ -41,6 +40,28 @@ The path of the temporary configuration file can be accessed inside
         def create_arguments(filename, file, config_file):
             return "--use-config", config_file
 
+If you return ``None``, no configuration file is generated. A common case
+where to explicitly return ``None`` is when the user wants to use his own
+tool-specific config. If a config file is supplied by user that means we
+don't have to generate a config file. Generating config file consumes
+resources. Avoiding generating config file increases the performance.
```

Comment:
~~If a config file ...~~
-->
```
In case the user specifies such a config file, we can avoid generating one again with ``generate_config`` to reduce I/O load.
```